operate/services/health_checker.py

```diff
@@ -180,7 +180,7 @@ async def _check_health(
                             f"[HEALTH_CHECKER] {service_config_id} not healthy for {fails} time in a row"
                         )
                     else:
-                        self.logger.info(
+                        self.logger.debug(
```

Comment:
We'll miss this in the logs, as we don't log debug info. Are we Ok with that?

Reply:
right, I removed it to not increase this log line by 5 times. As the helpful line would be the sad path, this is only for happy path.